tigerpath/majors_and_certificates/majors/PSY_2018.json

```diff
@@ -75,20 +80,20 @@
       "name": "Departmental Courses",
       "max_counted": 1,
       "min_needed": 8,
-      "explanation": "Psychology concentrators must successfully pass at least eight courses within the department in addition to the three prerequisite courses. If they were not already taken as prerequisites, PSY 252, PSY 255, and either PSY 258 or NEU 200 must be included in the eight courses. It is recommended that students finish these three courses by the end of junior year. One of the eight departmental courses must be PSY 300 (Research Methods in Psychology), which must be completed by the end of junior year. Of the remaining departmental courses, all must be 200-level or higher and at least four must be at the 300- or 400-level. Students can take up to two, pre-approved cognate courses from other departments, which count as 300-level-and-above departmental courses.",
+      "explanation": "The psychology concentration requires graded completion (without PDF) of at least eight courses within the department in addition to the three prerequisite courses. If they were not already taken as prerequisites, students must take PSY 252, PSY 255, and either PSY 258 or NEU 200. It is recommended that students finish these three courses by the end of junior year. One of the eight departmental courses must be PSY 300 (Research Methods in Psychology), which must be completed by the end of junior year. Of the remaining departmental courses, all must be 200-level or higher and at least four must be 300-level or higher. Students can take up to two cognate courses from other departments, approved by the Psychology Departmental Representative, to count as departmental courses.",
       "req_list": [
         {
           "name": "Required Departmentals",
           "max_counted": 4,
-          "min_needed": "ALL",
-          "explanation": "",
+          "min_needed": 4,
+          "explanation": "If they were not already taken as prerequisites, students must take PSY 252, PSY 255, and either PSY 258 or NEU 200. It is recommended that students finish these three courses by the end of junior year. One of the eight departmental courses must be PSY 300 (Research Methods in Psychology), which must be completed by the end of junior year.",
           "completed_by_semester": 6,
           "req_list": [
             {
               "name": null,
-              "max_counted": 3,
+              "max_counted": null,
               "min_needed": null,
-              "explanation": "",
+              "explanation": null,
               "course_list": [
                 "PSY 252",
                 "PSY 255",

```

Comment:
I understand that this was partly already weird before your changes, but what happens now if those three courses were already taken as prerequisites? Then this requirement wouldn't be satisfiable as it's coded here, right?

Reply:
I'm not quite sure I understand the concern. The way it's coded right now is that these courses also double count as fulfilling a prerequisite and the required departmentals.

![image](https://user-images.githubusercontent.com/37886745/86209182-92ca8d00-bba4-11ea-8bed-63dbb66b4e35.png)

Reply:
Yeah, you're right. I confused myself there.